Example/YelpAPI.xcodeproj/project.pbxproj

```diff
@@ -124,6 +128,8 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				0B5080851CBEEA950071508E /* OHHTTPStubs.framework in Frameworks */,
+				0B5080831CBEEA800071508E /* OCMock.framework in Frameworks */,
```

Comment:
This shouldn't be necessary, CocoaPods should handle this for us. Not sure why it failed before you added this, I'll see what I can recommend...